CMakeLists.txt

```diff
@@ -31,6 +31,9 @@ option(THREAD_SANITIZER "Build with ThreadSanitizer." OFF)
 option(UNDEFINED_BEHAVIOR_SANITIZER "Build with UndefinedBehaviorSanitizer." OFF)
 option(LINK_PROFILER "Link gperftools profiler" OFF)
 
+set(EXT_PTHREAD_INCLUDE_DIR "" CACHE PATH "Path to the PThread dir")
```

Comment:
This would be set only for Windows right? Can we add the platform check?

Reply:
Since we are allowing it to be provided from command line, I think it needs to exist like that. It is empty at first ("") and then when it is read from command line, it gets its value which is used later

Reply:
Its still a -D option right? If its other platforms, this blocks is just not visited and the variable is unused. So it should be ok to add this in the `if(WIN32)` block. It is more clear that way.

Reply:
Makes sense, done!